src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -164,10 +164,10 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
 
         # common to DockerBuildStep, FileTaskStep and RunTaskStep
         c.argument('context_path', options_list=['--context', '-c'], help="The full URL to the source code repository (Requires '.git' suffix for a github repo). If '/dev/null' is specified, the value will be set to None and ignored.")
-        c.argument('arg', help="Build argument in 'name[=value]' format.", action='append', validator=validate_arg)
-        c.argument('secret_arg', help="Secret build argument in 'name[=value]' format.", action='append', validator=validate_secret_arg)
-        c.argument('set_value', options_list=['--set'], help="Task value in 'name[=value]' format.", action='append', validator=validate_set)
-        c.argument('set_secret', help="Secret task value in 'name[=value]' format.", action='append', validator=validate_set_secret)
+        c.argument('arg', help="Build argument in 'name[=value]' format. Multiple --arg can be fed.", action='append', validator=validate_arg)
+        c.argument('secret_arg', help="Secret build argument in 'name[=value]' format. Multiple --secret-arg can be fed.", action='append', validator=validate_secret_arg)
+        c.argument('set_value', options_list=['--set'], help="Task value in 'name[=value]' format. Multiple --set can be fed.", action='append', validator=validate_set)
```

Comment:
help="Task value in '--set name[=value]' format. Multiples supported by passing --set multiple times.